src/week3/Ex4.java

```diff
@@ -0,0 +1,55 @@
+package week3;
+
+import java.util.Arrays;
+
+/**
+ * Created by Lilia Shamirian on  26, Sep, 2020
+ */
+public class Ex4 {
+    public static void main(String[] args) {
+        int[] array1 = new int[5];
+        int[] array2 = new int[5];
+
+        for (int i = 0; i < array1.length; i++) {
+            array1[i] = (int) (Math.random() * 10);
+        }
+
+        for (int i = 0; i < array2.length; i++) {
+            array2[i] = (int) (Math.random() * 10);
+        }
+
+        System.out.println("Array 1 : " + Arrays.toString(array1));
+        System.out.println("Array 2 : " + Arrays.toString(array2));
+        System.out.println("Combined Array Is : " + Arrays.toString(combineTwoArrays(array1, array2)));
+        System.out.println();
+        System.out.println("Sum of Two Arrays Average is : " + sumOfTwoArraysAverage(array1, array2));
+
+    }
+
+    private static int[] combineTwoArrays(int[] array1, int[] array2) {
+        int[] array3 = new int[array1.length + array2.length];
```

Comment:
your solution is correct!

but plaese try to get familiar with the Arrays copyof method to combine two arrays))

Reply:
ok thanks